### ISSUE TITLE
Remember me functionality

### ISSUE THREAD

**Issue by rics3n** [lynndylanhurley/ng-token-auth#107]
the only reference I found to to this functionality was in the style guide. Did I miss something or how would you deal with this? If you use the local storage option, the last auth token will be remembered beyond the session scope, but with the cookie option storing the expiry reply from the server would need to be set in the cookie options. Especially when you want to give the User the option between "remember me" and session scope auth, the cookie option would be my preferable choice. 

Thanks in advance for your thoughts on this.

**Comment by lynndylanhurley** [comment]
@rics3n - what do you mean by "remember me"?

**Comment by jugglingcats** [comment]
Many sites provide a "Remember Me" checkbox on the login form. If the user chooses this option their session token should be longer lived (days or weeks) rather than only browser session scope (ie. discarded when user closes down their browser).

**Comment by rics3n** [comment]
that is exactly what I mean. You can take a look at a possible implementation at my fork. I already took a shot at implementing it.

**Comment by lynndylanhurley** [comment]
> If the user chooses this option their session token should be longer lived (days or weeks) rather than only browser session scope (ie. discarded when user closes down their browser).

That is how it works currently. The tokens are saved indefinitely by the browser using either cookies or localStorage, and the API decides the duration of their validity. When using devise_token_auth, the default expiry is 2 weeks (see [token lifespan](https://github.com/lynndylanhurley/devise_token_auth#initializer-settings)).

**Comment by rics3n** [comment]
I indeed use it with devise_token_auth and it works great, except this functionality. The token on the serverside is sent correctly with expiry set to 2 weeks, but ng-token-auth is saving the cookie in session scope. So after closing the browser the token is deleted and the user has to login again. I only got it to work when setting the expiry date in ipCookie explicitly (e.g. ipCookie(key, value, { expires: 21 });). Otherwise ipCookie is alway saving Cookies in session scope.
